polaris/polaris/sep10/utils.py

```diff
@@ -4,31 +4,38 @@
 from jwt.exceptions import InvalidTokenError
 from rest_framework import status
 from rest_framework.request import Request
+from rest_framework.response import Response
 
 from polaris import settings
+from polaris.models import Transaction
 from polaris.utils import render_error_response
 
 
-def check_auth(request, func, content_type: str = "application/json"):
+def check_auth(request, func, sep, content_type: str = "application/json"):
     """
     Check SEP 10 authentication in a request.
     Else call the original view function.
     """
     try:
         account = validate_jwt_request(request)
     except ValueError as e:
-        return render_error_response(
-            str(e), content_type=content_type, status_code=status.HTTP_403_FORBIDDEN
-        )
+        if sep == Transaction.PROTOCOL.sep6:
```

Comment:
SEP-24 isn't opinionated about what the response body should look like for 403 permission denied responses. But SEP-6 explicitly gives the response body that should be returned: https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0006.md#5-authentication-required

Reply:
If SEP-24 isn't opinionated why not just always use the sep6 response?

Reply:
This isn't a strongly held opinion but I think using this response for SEP-24 doesn't really make as much sense. 

The `{'type': 'authentication_required'}` response makes sense in SEP-6 because its accompanying status code (403) is used for 3 other response types. Thats why SEP-6 has a `type` attribute in all 403 responses, to distinguish what kind of 403 response it is.

For all SEP-24 endpoints, error responses include an `error` key. If we make an exception for bad authentication responses clients may not be expecting a different schema. Again, this isn't a big deal but I think its better to be consistent.